packages/swap/src/helpers/orders/factories/makeOrderCalculations.ts

```diff
@@ -154,6 +154,19 @@ export const makeOrderCalculations = ({
       ]),
     }
 
+    const ptTotalValueSpent: Balance.Amount | undefined = Quantities.isZero(
```

Comment:
the reasoning behind Amounts to don't be undefined is to don't sum up (undefined + undefined) = 0, since here is a derivation of total PT, I'd say it's ok, but if its ever needed consider adding it to a sum, then we should remove the undefined

Reply:
shouldn't it always add the fees? maybe is a UI issue not state. dunno

Reply:
Problem here is that the price in ada can be missing, so all of this field needs to be undefined. Either all the value is there, or it doesn't represent the same thing.